lib/capistrano/tasks/database.rake

```diff
@@ -0,0 +1,109 @@
+namespace :forkcms do
+  namespace :database do
+
+    backup_file = 'mysql_backup.sql'
+
+    desc 'Create a backup of the database'
+    task :backup do
+      on roles(:web) do
+        # Fetch our flags to reach the database
+        flags = get_mysql_connection_flags
+
+        # Abort if no flags are found
+        if flags == nil
+          next
+        end
+
+        execute :mysqldump, "#{flags} --skip-lock-tables > #{release_path}/#{backup_file}"
+      end
+    end
+
+    desc 'Restore the database from a backup file'
+    task :restore do
+      on roles(:web) do
+        # Check if the file exists.
+        if capture("if [ -f #{release_path}/#{backup_file} ]; then echo 'yes'; fi").chomp != 'yes'
+          puts "No backup file found, create a backup first"
+          next
+        end
+
+        Rake::Task["forkcms:database:execute"].invoke("#{release_path}/#{backup_file}")
```

Comment:
Moet ge dan niet zeker zijn dat er ook drop-statements in staan?

Reply:
drop tables is a default mysqldump option